### PR TITLE
Add IPFS hash of wasm to the subgraph.yaml file

### PR DESCRIPTION
**Pull request by thala321**

### CODE REVIEW

**Comment by thala321** [comment]
Screenshot from 2021-08-30 15-30-02

File: code-compiler/compiler/wasm.py [hunk 1]
```diff
@@ -145,7 +130,53 @@ def replace_ipfs_hash(subgraph_type, subgraph, abi_res):
         for i in range(0, len(subgraph[subgraph_type][0]['mapping']['abis'])):
             file_name = os.path.basename(subgraph[subgraph_type][0]['mapping']['abis'][i]['file'])
             name = subgraph[subgraph_type][0]['mapping']['abis'][i]['name']
-            for abi in abi_res:
-                if file_name.lower() == abi["name"].lower():
-                    subgraph[subgraph_type][0]['mapping']['abis'][i] = {'name': name, 'file': {'/': '/ipfs/' + abi["hash"]}}
+            for abi_object in abi_res:
```
Comment (anhhuy0501) [inline code-compiler/compiler/wasm.py:133]: Refactor create abi dictionary by project.yaml for safer code.

Comment (thala321) [reply code-compiler/compiler/wasm.py:133]: https://github.com/massbitprotocol/massbitprotocol/issues/180

**Review by anhhuy0501 — APPROVED** [review]
LGTM

**Pull request merged**